packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -92,33 +103,44 @@ storiesOf('Patterns (Sub-Patterns)|Card', module)
       '1:1': '1x1',
     };
 
-    return !select('Ratio', ratio, ratio['none']) ? (
-      <Card
-        title={title}
-        copy={copy}
-        href={href}
-        cta={cta}
-        icon={ArrowRight20}
-        target={target}
-        type={type}
-      />
-    ) : (
+    const themes = {
+      white: '',
+      g10: 'g10',
+      g90: 'g90',
+      g100: 'g100',
+    };
+
+    const theme = select('theme', themes, themes.g100);
+
+    const type = text('type', 'link');
```

Comment:
is this repeated from line `18`?

Reply:
whoops, yeah removing from line 18 as I want the type to be different between the 2 stories